projects/configs/tag-operations/manufacturer-overrides/mr-spectroscopy.yaml

```diff
@@ -0,0 +1,44 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+- manufacturer: ^siemens
```

Comment:
do we want to keep all the private tags?

Reply:
I'd probably not keep them in the first instance, as we don't have documentation of what they are going to have in them. We can add in any fields if we have researchers say that we need them

Reply:
okay cool, I'll remove them for now